force-app/main/default/classes/ContactController.cls

```diff
@@ -7,9 +7,6 @@ public with sharing class ContactController {
 
     @AuraEnabled(cacheable=true)
     public static List<Contact> findContacts(String searchKey) {
```

Comment:
Removing this will break `apexWireMethodWithParams` and `apexImperativeMethodWithParams` as both will then return all contacts if the search field is empty.

Reply:
I don't think it's breaking these recipes. It's just a different (and I think more logical) implementation: If you don't provide a filter value, you get the unfiltered list (with a LIMIT of course). In other words, we start with a default list (could be fined tuned to be MRU) instead of an empty list which seems to be the pattern currently used in many searchable list-type UIs. If we want an empty list when the component loads then we shouldn't invoke the method at all.

Reply:
Yeah, I see. Although on first invocation all recipes need to have an initial value. Currently they are `undefined`. That means the user doesn’t get on first call the full list, but only later after s/he cleared the first search. That was the reason why I added the `isBlank` check earlier. ;-)